### PR TITLE
arg attributes in MathML examples in open concept list

### PR DESCRIPTION
**Pull request by davidcarlisle**
This adds `arg` attributes to most MathML examples (and a single test `intent` attribute) and has some styling using @dginev 's version of the css for arg.

It's work in progress but I'd like to merge anyway to avoid getting too far ahead on a branch, and give time to bring the core list to the same format.

The html/css rendering of the arg attributes is visible by hovering over them at

https://davidcarlisle.github.io/mathml-docs/intent-open-concepts/

### CODE REVIEW

File: _data/open.yml
Comment (dginev) [inline _data/open.yml:913]: For bijection the example with `f` was poorly chosen by me - predates properties. It would be `f:bijection` in a world where `is-a` property use was still available.

A better example would be an infix arrow, but those are non-standard, [math SE discussion](https://math.stackexchange.com/questions/46678/what-are-usual-notations-for-surjective-injective-and-bijective-functions). Maybe `X ↔ Y` with the `↔` marked as `intent="bijection"`, maybe not.

Comment (davidcarlisle) [reply _data/open.yml:913]: Not sure why you say infix arrows are non standard, all the arrows are infix binary by default. Oh you mean people on that SE thread don't agree on which arrows to use, not that arrows being infix are non standard, OK.

But yes, in general lots of the examples will need to be reviewed there are several I've spotted while doing this round that I plan to raise issues on after the merge.

But I'm trying to keep this PR  just on the format so replacing `notation:` field with mathml hints to actual mathml. Current status is 1258 `<math>` elements and 248 `notation[a-z]:` fields so it's getting there...

But if someone (hint) would approve the merge I could merge this bulk edit to w3c, and then individual issues and/or PR  could be raised on problematic examples.

As it is, editing the w3c copy will lead to merge conflicts and only I have edit writes on my fork so I really would rather forks are short lived.

Comment (dginev) [reply _data/open.yml:911]: The bijection change was made in this PR, hence I reviewed it here. If anything, I have been trying to keep my comments to a helpful minimum.

I will pass my review flag to Moritz @physikerwelt who - if interested - may help resolve the PR. I am not comfortable doing much more here.

Comment (davidcarlisle) [reply _data/open.yml:911]: as you wish, I wouldn't have said it's a change here Ijust added mathml if we decide to treat this as a  property rather than a concept the whole row wiuld presumably go, but that's fine there will neeed to be several such edits whatever format the notation is shown. mathml with or without tintent attributes.

File: intent-open-concepts/index.md
Comment (physikerwelt) [inline intent-open-concepts/index.md:8]: Where will this div element put. (I recently learned that it is often better to puth mathml elements in a span)

File: _data/open.yml
Comment (physikerwelt) [reply _data/open.yml:911]: My comments:
- Can we have intent attributes on the mathml root element, that way we would not need to add an mrow?
- I think the overall pattern a1-9 is good.
- Can I check the result of this PR somewhere visually?

File: intent-open-concepts/index.md
Comment (davidcarlisle) [reply intent-open-concepts/index.md:8]: @physikerwelt  so it's not putting the math element in a div it is removing the math element entirely.

File: _data/open.yml
Comment (davidcarlisle) [reply _data/open.yml:911]: @physikerwelt  we could have intent om math but i added the mrow as if you ever want to coy these fragments into a larger expression  you would been to habve mrows so it seemed simpler to add them even though it does add one extra layer in these smal examples. Yoes the fork is rendering at  https://davidcarlisle.github.io/mathml-docs/intent-open-concepts/ including the button to show mathml source. (Currently you need to refresh the page to get back to the initial state)

Comment (physikerwelt) [reply _data/open.yml:911]: Thank you. This looks good. Two minor issues:

- Invalid markup: Incorrect number of children for <msup/> tag.
- “mathvariant='bold'” on MathML elements is deprecated and will be removed at a future date.

File: _data/open.yml
Comment (physikerwelt) [inline _data/open.yml:8220]: you could replace the n with 𝐧 (U+1D427 (119847))?

Comment (davidcarlisle) [reply _data/open.yml]: @physikerwelt  I  seemed to have two n there. I  made one an m but would you really use bold upright not just normal math italic?  but in general as I commented to Deyan I'd rather reviewing individual entries happens at w3c with more reasonable sized diffs as this PR was really asking about the change in format to add arg= attributes which is nearly completed now just 126 rows with `!!F` and no real mathml out of 1264 math elements. But diff for this PR is now nearly 5000 lines I want to merge as is,  to allow individual entries to be reviewed/edited at w3c (where more people have write access)

File: _data/open.yml
Comment (physikerwelt) [inline _data/open.yml:8220]: I see that a lot of lines were changed. However this is the only line that causes a warning in Firefox. Thus I was thinking it would be good to come from 1 to 0 warning. I have no particular feeling about the unicode character.

**Review by physikerwelt — APPROVED** [review]
I am happy if this is merged. However, I think it would be better if we could fix the warnings generated in the browser console.

File: _data/open.yml
Comment (davidcarlisle) [reply _data/open.yml:8220]: @physikerwelt  ah sorry yes the warning is the mathvariant bold which isn't actually valid i added it as unicode doesn't have bold brackets I;'ll replace bycss. Firefox also tells me Invalid markup: Incorrect number of children for <msup/> tag.`  update coming up....

File: _data/open.yml
Comment (dginev) [reply _data/open.yml:911]: Thank you Moritz for the time reviewing the PR. 

I would suggest to David to tag Moritz directly in future PRs as the outcomes appear more productive.

File: _data/open.yml
Comment (davidcarlisle) [reply _data/open.yml]: @physikerwelt  thanks, fixed

**Comment by davidcarlisle** [comment]
> I am happy if this is merged. However, I think it would be better if we could fix the warnings generated in the browser console.

yes sorry about that firefox console is now clean

**Pull request merged**
